frontend/package.json

```diff
@@ -42,8 +42,8 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.24.1",
-    "@typescript-eslint/eslint-plugin": "^6.0.0",
-    "@typescript-eslint/parser": "^6.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.5.0",
+    "@typescript-eslint/parser": "^5.5.0",
```

Comment:
Is there any reason these were downgraded?

Reply:
Whoops! That was unintended, I have re-added those.